### ISSUE TITLE
With `Build before ReAttaching` enabled if the build fails don't attach to process.

### ISSUE THREAD

**Issue by cyclr-adrian** [erlandranvinge/ReAttach#27]
I'm using VS 2022 (Version 17.1.6) and have installed the ReAttach VSIX extension and have enabled the option `Build before ReAttaching`.

When I ReAttach using the extension my solution builds and then re-attaches to the last process which is what I expected it to do.

However if I make a change which stops the solution from successfully compiling the ReAttach extension runs the build as before but the build correctly fails but the ReAttach to the process still happens.  I would have expected on a failed build that the ReAttach to process would not occur, highlighting an issue.

**Comment by erlandranvinge** [comment]
I totally agree. Apparently, that scenario was never tested. In fact, when looking at the code, the result of the actual build is not taken into account at all.

**Comment by erlandranvinge** [comment]
It seems like the automation object used to trigger build doesn't return anything. I guess one would need to implement something a bit more sophisticated in order to address this. E.g. IVsSolutionBuildManager might hold the key.
